JavaScript/d-messenger/.eslintrc.json

```diff
@@ -135,10 +139,6 @@
       "error",
       "always"
     ],
-    "operator-linebreak": [
-      "error",
-      "after"
-    ],
```

Comment:
```js
    "operator-linebreak": [
      "error",
      "after",
      { "overrides": { "?": "before", ":": "before" } }
    ],
```

Reply:
Thank you for the review and suggesting a lot better decision! I had implemented it in the latest commit.